README.rst

```diff
@@ -0,0 +1,43 @@
+django-ipam
+=================
+
+.. image:: https://travis-ci.org/openwisp/django-ipam.svg
+   :target: https://travis-ci.org/openwisp/django-ipam
+
+.. image:: https://coveralls.io/repos/openwisp/django-ipam/badge.svg
+  :target: https://coveralls.io/r/openwisp/django-ipam
+
+------------
+
+Django-ipam is part of the `OpenWISP project <http://openwisp.org>`_.
+
+.. image:: http://netjsonconfig.openwisp.org/en/latest/_images/openwisp.org.svg
+  :target: http://openwisp.org
+
+**django-ipam** is Django reusable app for IP Address Management.
+
+------------
+
+How to Setup ?
+================
+Install your cloned fork:
+
+.. code-block:: shell
+
+    git clone https://github.com/openwisp/django-ipam.git
+    cd django-ipam
+    pip install -r requirements.txt
+    python setup.py develop
```

Comment:
Doesn't `python setup.py develop` install requirements as well?

Reply:
My bad, I forgot about that. Thank you for pointing it out :+1: